core/tests/webdriverio_desktop/explorationHistoryTab.js

```diff
@@ -722,6 +722,9 @@ describe('Exploration history', function () {
     // Revert to version 2.
     await explorationEditorPage.navigateToHistoryTab();
     await explorationEditorHistoryTab.revertToVersion(2);
+    // Due to the angular migration, the system navigates to the main tab
+    // too quickly to check for the revert message and version. Hence,
+    // we need to renavigate to the history tab to check for those.
```

Comment:
@mon4our Can you explain how angular migration is responsible for this?

Reply:
AngularJS and Angular have different ways of handling page reloads. Angular on reload using `this.windowRef.nativeWindow.location.reload()` navigates to the default page (main tab). I went through a few StackOverflow discussions regarding this and they suggest to use the angular router for reloads which is not possible in our case as the entire exploration editor router service uses `windowRef.nativeWindow.location`.